bigbluebutton-html5/imports/ui/services/audio-manager/index.js

```diff
@@ -566,6 +575,28 @@ class AudioManager {
       this.autoplayBlocked = true;
     }
   }
+
+  mute () {
+    const bridge = (this.useKurento && this.isListenOnly) ? this.listenOnlyBridge : this.bridge;
+    const peer = bridge.getPeerConnection();
+    peer.getSenders().forEach(sender => {
+      const { track } = sender;
+      if (track && track.kind === 'audio') {
+        track.enabled = false;
+      }
+    });
+  }
+
+  unmute () {
+    const bridge = (this.useKurento && this.isListenOnly) ? this.listenOnlyBridge : this.bridge;
+    const peer = bridge.getPeerConnection();
+    peer.getSenders().forEach(sender => {
+      const { track } = sender;
+      if (track && track.kind === 'audio') {
+        track.enabled = true;
+      }
+    });
+  }
```

Comment:
Besides one line the functions are identical, and could be refactored.

Reply:
Done, thanks for the ping (#10433)